docs/transaction-api.md

```diff
@@ -4,84 +4,83 @@
 
 ### Payment creation
 
-A user creates a payment by passing a `PaymentIntent` to `Mutation.createOutgoingPayment`. If the payment destination (the payment pointer or invoice URL) is successfully resolved, the payment is created in the `QUOTING` state.
+A user creates a payment by passing a `PaymentIntent` to `Mutation.createOutgoingPayment`. If the payment destination (the payment pointer or incoming payment URL) is successfully resolved, the payment is created in the `PENDING` state.
```

Comment:
Though on that subject; what is the rationale for creating a (failed) payment when the destination is invalid, rather than just responding with an error from Mutation.createOutgoingPayment?

[Adrian had said](https://hackmd.io/YWnLmu9ASiiHIwTPzmZKNg?view#3-Outgoing-Payment):
>I think that's a lot of synchronous processing to do while the client waits for the response.

With #252, should this still refer to `Mutation.createOutgoingPayment`? Should the graphql admin api still include `Mutation.createOutgoingPayment` or any other requests available via open payments endpoints?